### ISSUE TITLE
Charge Warrior GENERAL

### ISSUE THREAD

**Issue by neano300** [Valhalla-Gaming/Tracker#8820]
**Describe the issue you're having**:
This spell fall the 90% of the times when you use it.
**Explain how you expect it work**:
This spell CAN'T fall.
**Steps to reproduce the problem**:

1.  Charge
2.  Lots of times never charge the enemy.

**Links to Wowhead, YouTube, etc**:
http://www.wowhead.com/spell=100/charge
http://imgur.com/a/3gPK2

This is a huge problem for warriors because the pvp is really unbalanced and if we can't start with our main spell then delete the warrior from DB.

**Comment by Nightcap** [comment]
yup, makes you fall through the world and die very often, on uneven terrain. and fails on use often, pathing is buggy
